src/bp/ui-studio/src/web/views/OneFlow/diagram/ConditionForm/index.tsx

```diff
@@ -76,6 +80,9 @@ const ConditionForm: FC<Props> = ({
   ]
 
   const handleConditionChange = value => {
+    if (!['user_intent_is', 'raw_js'].includes(value)) {
+      document.documentElement.style.setProperty('--right-sidebar-width', '240px')
```

Comment:
This state should be moved to Redux. There's a couple of issues with the right sidebar width right now, especially related to the webchat panel.

- If the webchat is opened, when you click to open a trigger (or any other form) you will not see it (but the space will be filled in gray)
![image](https://user-images.githubusercontent.com/42552874/92521811-df0cfb80-f1eb-11ea-96d5-59f79e8fe058.png)
- If you open the webchat while on another module (ex: analytics), and then go to the flow builder, you get this
![image](https://user-images.githubusercontent.com/42552874/92521884-f815ac80-f1eb-11ea-96c4-6bc05fc65192.png)
 instead of this 
![image](https://user-images.githubusercontent.com/42552874/92521899-fd72f700-f1eb-11ea-8508-98f6881e50df.png)

Reply:
I'll fix it in a different PR just so we can merge this one, I'll open a ticket so I don't forget

Reply:
https://trello.com/c/DYt8vumh/366-move-documentdocumentelementstylesetproperty-right-sidebar-width-240px-to-redux-store

Reply:
Then you need to comment the code for https://trello.com/c/QFDrwAmS/350-inspector-panel-should-not-be-expanded-when-the-expandable-feature-isnt-available because it's breaking the studio, we simply cannot use the inspector while the webchat is open

Reply:
I only thing done on this PR in regard to the expand feature is to remove the expanded state if the selected option doesn't have the expand option... The issues were there before this PR so I would just merge this one and then start a new one right after this is merged to address those issues and have one PR focusing only on that. It's better for the change logs, for testing and keep the PRs open for a shorter amount of time that way

Reply:
nvm was quicker than expected and no one was online so I fixed it in this one